crates/db_views/src/post_view.rs

```diff
@@ -358,6 +358,20 @@ impl<'a> PostQueryBuilder<'a> {
         .then_order_by(post_aggregates::stickied.desc());
     }
 
+    // Unless on a community's page or if subscribed to a community, filter out blacklisted communities from post
```

Comment:
I do remember running it through some manual test cases. I can upload a video next time to be sure.

Reply:
> And since you're only filtering things from the subscribed view, and not All, I don't see how you could've tested this.

Now I see what I did, for subscribed I did nothing for everything not subscribed I filtered out. Because it wouldn't show up in subscribed unless subscribed. Maybe I can make that purpose a little more clear.